tool/webqa-update-canonicals.js

```diff
@@ -56,6 +56,12 @@ module.exports = {
             help: 'Skip the entire process.',
             defaultValue: false
         });
+        parser.addArgument('--no-neural', {
```

Comment:
This would be more understandable as `--only-heuristics`, or even better `--algorithm={heuristic,neural}`

Reply:
updated with c3aa6e0